network-api/networkapi/campaign/views.py

```diff
@@ -12,6 +12,12 @@
 from networkapi.wagtailpages.models import Petition, Signup
 
 
+def process_lang_code(lang):
+    # Salesforce expects "pt" instead of "pt-BR".
+    # See https://github.com/mozilla/foundation.mozilla.org/issues/5993
+    return lang if lang != 'pt-BR' else 'pt',
```

Comment:
small tweak, but:
```suggestion
    if lang == 'pt-BR':
        return 'pt'
    return lang
```
mostly to make it easy to etend this for more languages should we need to in the future.